guide.md

```diff
@@ -12392,7 +12517,7 @@ You can find some introduction on these on these projects:
 
 [^124]: Wikipedia, Google SensorVault, <https://en.wikipedia.org/wiki/Sensorvault> <sup>[[Wikiless]][898]</sup> <sup>[[Archive.org]][899]</sup>
 
-[^125]: NRKBeta, My Phone Was Spying on Me, so I Tracked Down the Surveillants <https://nrkbeta.no/2020/12/03/my-phone-was-spying-on-me-so-i-tracked-down-the-surveillants/> <sup>[[Archive.org]][900]</sup>
+[^125]: NRKBeta, My Phone Was Spying on Me, soWeTracked Down the Surveillants <https://nrkbeta.no/2020/12/03/my-phone-was-spying-on-me-so-i-tracked-down-the-surveillants/> <sup>[[Archive.org]][900]</sup>

```

Comment:
Should be "So We Tracked", with beginning of each word capitalized to match the rest of the title.